examples/step-3/doc/intro.dox

```diff
@@ -102,19 +111,24 @@ where the matrix $A$ and the right hand side $F$ are defined as
   \\
   F_i &= (\varphi_i, f).
 @f}
+
+
+<h3> Multiplying with a test function from the left or from the right? </h3>
```

Comment:
If you don't mind (I hope that I am not becoming irritating with all of these grammatical comments, ultimately its not that important) this sentence is missing some words: it should be "Should we multiply by a test function on the left or right?" or something similar, or (simpler) we could just drop the question mark.